tests/dump/alias.wat

```diff
@@ -3,6 +3,10 @@
     (export "f1" (func $f1))
     (export "f2" (func $f2 (param i32)))
   ))
+
+  ;; TODO figure out syntactic sugar here
+  (alias $i "f1" (func $i.$f1))
+
   (func (export "run")
     call $i.$f1
   )
```

Comment:
Soon! I haven't implemented these changes in the wat format yet, but I plan to do so after this PR. It seemed like an easily separable change since it's just syntax sugar in the text format.